e2e/init.js

```diff
@@ -1,16 +1,25 @@
 const detox = require('detox');
 const config = require('../package.json').detox;
 const exec = require('shell-utils').exec;
+jest.setTimeout(480000);
```

Comment:
Is such a long timeout really needed?

Reply:
We can lower this, nothing smart behind this number..